### PR TITLE
Can now load sections that are larger than calls and label them as calls

### PR DESCRIPTION
**Pull request by noamzbr**
Also, returned data equalization. why did we remove it?

### CODE REVIEW

**Comment by Z30G0D** [comment]
> Also, returned data equalization. why did we remove it?

The equalizing distribution (or resampling actually) is now part of `train.py`.
The `DataLoader` is responsible for this operation and not a method of `DataSet`.

File: soundbay/data.py [hunk 1]
```diff
@@ -220,25 +244,42 @@ def _get_audio(self, path_to_file, begin_time, end_time, label, channel=None):
...
-            else:
-                start_time = random.randint(begin_time, end_time - int(self.seq_length * self.data_sample_rate))
-            if start_time < 0:
-                start_time = 0
+        seg_length = end_time - begin_time
+        requested_seq_length = int(self.seq_length * self.data_sample_rate)
+        last_start_time = sf.info(path_to_file).frames - requested_seq_length
+        # Do all this stuff only to calls in training set, because otherwise _slice_sequence has already been done
+        if self.mode == "train":
+            if seg_length > requested_seq_length:
```
Comment (mosheman5) [inline soundbay/data.py]: The flow looks legit, good bug catch with the start time of label==0!

File: tests/test_data.py [hunk 2]
```diff
@@ -14,7 +14,7 @@ def test_dataloader() -> None:
                                         augmentations=cfg.data.train_dataset.augmentations,
                                         augmentations_p=cfg.data.train_dataset.augmentations_p,
                                         preprocessors=cfg.data.train_dataset.preprocessors)
-        assert data_loader.metadata.shape[1] == 5  # make sure metadata has 5 columns
+        assert data_loader.metadata.shape[1] in (5, 6)  # make sure metadata has 5/6 columns (account for channel)
```
Comment (mosheman5) [inline tests/test_data.py:17]: like for maintaining the tests

**Review by mosheman5 — APPROVED** [review]
Ahla feature
